app/controllers/api/v1/donation_tokens_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::DonationTokenssController < Api::V1::BaseController
```

Comment:
If we just want to see if the token is still valid, what do you think about letting the client make a HEAD request? We could 410 back if the token is expired and the client to handle that condition?